docs/maui/views/RatingView.md

```diff
@@ -0,0 +1,519 @@
+}
+```
+
+## Set shape
+The `Shape` property is an `enum` of type `RatingViewShape` for setting the rating item shape of the rating items, such as stars, circles, like, dislike, or any other commonly used rating icons..  The available options are:
+
+- `Star` - (default)
+- `Heart`
+- `Circle`
+- `Like`
+- `Dislike`
+- `Custom` - If set and `CustomShape` is NULL or empty, defaults to `Star`
+
+The following examples set the rating fill property:
+
+```xaml
+<toolkit:RatingView
+	Shape="Star" />
+<toolkit:RatingView
+	Shape="Heart" />
+<toolkit:RatingView
+	Shape="Circle" />
+<toolkit:RatingView
+	Shape="Like" />
+<toolkit:RatingView
+	Shape="Dislike" />
+<toolkit:RatingView
+	Shape="Custom" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView starRatingView = new()
+{
+	Shape = RatingViewShape.Star,
+};
+RatingView heartRatingView = new()
+{
+	Shape = RatingViewShape.Heart,
+};
+RatingView circleRatingView = new()
+{
+	Shape = RatingViewShape.Circle,
+};
+RatingView likeRatingView = new()
+{
+	Shape = RatingViewShape.Like,
+};
+RatingView dislikeRatingView = new()
+{
+	Shape = RatingViewShape.Dislike,
+};
+RatingView customRatingView = new()
+{
+	Shape = RatingViewShape.Custom,
+};
+```
+
+## Set shape border color
+The `ShapeBorderColor` is a `Color` for setting the border color of the rating item shape. This provides additional flexibility to create visually distinct and stylized rating shapes with custom borders.
+
+The following examples set the shape border color property:
+
+```xaml
+<toolkit:RatingView
+	ShapeBorderColor="Grey" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	ShapeBorderColor = Colors.Grey,
+};
+```
+
+## Set shape border thickness
+The `ShapeBorderThickness` is a `double` for setting the thickness of the shape border. This provides additional flexibility to create visually distinct and stylized rating shapes with custom borders.
+
+The following examples set the shape border thickness property:
+
+```xaml
+<toolkit:RatingView
+	ShapeBorderThickness="3" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	ShapeBorderThickness = 3,
+};
+```
+
+## Set spacing
+The `Spacing` is a `double` for setting the spacing between each rating item.
+
+The following examples set the shape border thickness property:
+
+```xaml
+<toolkit:RatingView
+	Spacing="7" />
+```
+
+The equivalent C# code is:
+
+```csharp
+RatingView ratingView = new()
+{
+	Spacing = 7,
+};
+```
+
+
+## Examples
+
+You can find examples of this control in action in the .NET MAUI Community Toolkit Sample Application:
+- [XAML Syntax](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/RatingView/RatingViewXamlPage.xaml)
+- [C# Syntax](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Views/RatingView/RatingViewCsharpPage.xaml)
```

Comment:
It's possibly not important but I find it intriguing that the Csharp page example is a xaml file, is this correct?

Reply:
Copy paste issue here, where I failed to update the extension.  Will fix.